src/main/java/kanbanflow/ui/pages/Board.java

```diff
@@ -146,67 +148,175 @@ public boolean isPendingInvitations() {
+     */
     public boolean isTaskInBoard(String Description) {
         return isInBoard.getText().equals(Description);
     }
 
-    /**
-     * Method delete task that was create.
-     */
     @FindBy(xpath = "//div[contains(@class,'task task-yellow')]")
     private WebElement taskInBoard;
     @FindBy(xpath = "//button[contains(@class,'taskDetails-sidebar-deleteButton taskDetails-sidebarButton')]")
     private WebElement trashTask;
     @FindBy(xpath = "//button[contains(@class,'modalBox-button button button--danger')]")
     private WebElement buttonOkTrash;
 
+    /**
+     * Method delete task that was create.
+     */
     public void Deletetask() {
         taskInBoard.click();
         trashTask.click();
         buttonOkTrash.click();
     }
 
-    /**
-     * Method compare if member is already.
-     */
     @FindBy(xpath = "//div[contains(@class, 'modalBox-body scroll scroll--rounded')]")
     private WebElement messageIfexist;
 
+    /**
+     * Method compare if member is already.
+     */
     public boolean isAlreadyMemberExist() {
         return messageIfexist.getText().equals(" is already a member of the board");
     }
+
+    @FindBy(xpath = "//table[@id='board-table']/tbody/tr/td/div/div[2]/div/span")
+    private WebElement taskCath;
+    @FindBy(xpath = "//table[@id='board-table']/tbody/tr/td[2]/div/div[2]")
+    private WebElement taskDropToday;
+    @FindBy(xpath = "//table[@id='board-table']/tbody/tr/td[3]/div/div[2]")
+    private WebElement taskDropInProgress;
+    @FindBy(xpath = "//table[@id='board-table']/tbody/tr/td[4]/div/div[2]")
+    private WebElement taskDropDone;
+
+    /**
+     * Let move a task to different fields in the board.
+     */
+    public void moveTask() {
+        Actions builder = new Actions(WebDriverManager.getInstance().getDriver());
+        builder.dragAndDrop(taskCath, taskDropToday).perform();
+        builder.dragAndDrop(taskCath, taskDropInProgress).perform();
+        builder.dragAndDrop(taskCath, taskDropDone).perform();
+        builder.build();
+    }
+
+    /**
+     * Compare if exist a element in the field done.
+     *
+     * @return boolean value.
+     */
+    public boolean isInDoneTask() {
+        return taskDropDone.isDisplayed();
+    }
+
+    @FindBy(xpath = "//div[@id='header']/div/button/i")
+    private WebElement buttonBoardMenu;
+    @FindBy(xpath = "//a[contains(text(),'Create board')]")
+    private WebElement createBoardButton;
+    @FindBy(xpath = "//input[@id='createBoard-boardName']")
+    private WebElement boardName;
+    @FindBy(xpath = "//button[contains(.,'Next')]")
+    private WebElement buttonNext;
+    @FindBy(xpath = "//button[contains(.,'Create board')]")
+    private WebElement buttonCreateBoard;
+
+    /**
+     * Method let create a new board.
+     */
+    public void newBoard(String nameBoard) {
+        buttonBoardMenu.click();
+        createBoardButton.click();
+        boardName.sendKeys("nameBoard");
+        buttonNext.click();
```

Comment:
All these clickied actions must be in one unique parameter, I think it's not right to repeat it this way.